pom.xml

```diff
@@ -51,15 +51,40 @@
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <ramlfiles_util_path>${ramlfiles_path}/raml-util</ramlfiles_util_path>
     <ramlfiles_acq_models_path>${ramlfiles_path}/acq-models</ramlfiles_acq_models_path>
-    <raml-module-builder.version>35.0.4</raml-module-builder.version>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <raml-module-builder.version>35.1.0</raml-module-builder.version>
+    <project.build.sourceEncoding>UTF-8 </project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+    <!--Dependencies-->
     <mod-configuration-client.version>5.9.1</mod-configuration-client.version>
+    <folio-di-support.version>2.0.1</folio-di-support.version>
+    <spring.version>6.0.2</spring.version>
+    <vertx.version>4.4.5</vertx.version>
+    <log4j.version>2.20.0</log4j.version>
+    <aspectj.version>1.9.20.1</aspectj.version>
+    <rest-assured.version>5.3.2</rest-assured.version>
+    <build-helper-maven-plugin.version>3.4.0</build-helper-maven-plugin.version>
+    <aspectj-maven-plugin.version>1.13.1</aspectj-maven-plugin.version>
+
+    <!--Maven dependencies-->
```

Comment:
propose to rename to maven plugin dependenices

Reply:
Renamed